README.md

```diff
@@ -27,11 +27,25 @@ Build the site with drush make:
 % drush make --contrib-destination=profiles/ding2/ drupal.make web --working-copy
 ```
 
-Then you can start the Docker containers:
+### Then you can start the Docker containers:
+
+If you're using docker-sync for Mac, you can use:
+
+```sh
+% docker-sync-stack start
+```
+
+otherwise, you can use vanilla docker-compose:
+
 ```sh
 % docker-compose up
 ```
 
+or if you are using Mac, you can use docker-sync
+```
+% docker-sync-stack start
```

Comment:
This seems to be mentioned twice. Is that intentional?

Reply:
Nope it wasnt - I've removed it now